packages/suite/src/hooks/wallet/useCoinmarketSavingsSetup.ts

```diff
@@ -201,10 +201,7 @@ export const useSavingsSetup = ({
         async (formValues: SavingsSetupFormState) => {
             const { customFiatAmount, fiatAmount, paymentFrequency, country } = formValues;
             if (selectedProvider && country) {
-                const result = await openCoinmarketSavingsConfirmModal(
```

Comment:
nit:
I think this would be better if a better name than `result` is chosen.

Reply:
This code is actually being removed. 😉

Reply:
? You moved it to if (condition)

Reply:
I wanted to say, that it was better like this. However, a better name for `result` would be nice.

```
const isSavingsModalConfirmed = await openCoinmarketSavingsConfirmModal(
    selectedProvider.companyName,
);
if (isSavingsModalConfirmed) {
```